quaddtype/numpy_quaddtype/_quaddtype_main.pyi

```diff
@@ -81,6 +81,12 @@ class QuadPrecision:
     # See https://github.com/python/mypy/issues/18343#issuecomment-2571784915
     def __new__(cls, /, value: _IntoQuad, backend: _Backend = "sleef") -> Self: ...
 
+    # Attributes
```

Comment:
These new attributes aren't added in stubs for the PR #208 , small addition do did in this PR